http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -1165,14 +1299,29 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
           )
           .transaction
 
-        dbName <- createDatabaseInWsm(app,
+        dbNamePrefixes = app.appType match {
```

Comment:
Generalized to a list of database prefixes rather than hardcoded wds, led to some tricky stuff as far as mapping the (async) database creation across the list.